.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+### Issue details
+
+_Please provide issue details here_.
+
+### Steps to reproduce/test case
+
+_Please provide necessary steps for reproduction of this issue, or better the
+reduced test case (without any external dependencies, if possible)_.
```

Comment:
maybe add: If we can't reproduce the issue from the description, we will probably have to ask for this, so best provide it up front

^--- some kind of warning that this isn't random time wasting... if the info isn't there, they'll just be asked later.